.spi.yml

```diff
@@ -2,4 +2,4 @@ version: 1
 builder:
   configs:
   - scheme: Sharing
-  - documentation_targets: [Sharing]
+    documentation_targets: [Sharing]
```

Comment:
If this is an array it ends up with two configs and then the lookup by platform/Swift version will only find the first one (given that both are essentially "wildcard" configs in that they don't specify any constraints). I.e. the first one will always "win".